tests/test_enumeration.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import pytest
+import xml.etree.ElementTree as et
+import lsst.ts.xml as ts_xml
+
+
+@pytest.mark.parametrize("xmlfile,csc,topic", ts_xml.get_xmlfile_csc_topic())
+def test_enumeration(xmlfile, csc, topic):
+    """Test that enumerations have the correct format.
+
+    Parameters
+    ----------
+    xmlfile : `pathlib.Path`
+        Full filepath to the Events XML file for the CSC.
+    csc : `testutils.subsystems`
+        Name of the CSC
+    """
+    # Test the topic <EFDB_Name> field.
+    with open(str(xmlfile), "r", encoding="utf-8") as f:
+        tree = et.parse(f)
+    root = tree.getroot()
+
+    for sal_enum in root.findall("Enumeration"):
+        for enum_value in sal_enum.text.split(','):
+            assert len(enum_value.rstrip().lstrip()) > 0, f"Invalid enumeration value in {csc}/{topic}."
```

Comment:
Consider `enum_value.strip()` instead of rstrip followed by lstrip.

I strongly suggest using a regex to test the stripped values, probably: `[A-Z][a-zA-Z0-9_]*`.